Go/Go.sublime-syntax

```diff
@@ -401,7 +401,7 @@ contexts:
 
   match-imaginary:
     # Mandatory integer, optional fraction, optional exponent, mandatory imaginary
-    - match: \d+(?:(\.)\d+)?(?:({{exponent}})\d+)?(i)
+    - match: \d+(?:_\d+)*(?:(\.)\d+(?:_\d+)*)?(?:({{exponent}})\d+(?:_\d+)*)?(i)
       scope: constant.numeric.imaginary.go
```

Comment:
According to scope naming guidelines `constant.numeric.imaginary.go` would need to be renamed to `constant.numeric.complex.imaginary.go`

Reply:
✔️ Renamed.